src/main/java/com/fasterxml/jackson/core/JsonFactory.java

```diff
@@ -1056,6 +1071,21 @@ public JsonFactory setCodec(ObjectCodec oc) {
 
     public ObjectCodec getCodec() { return _objectCodec; }
 
+    /**
+     * @param maxErrorTokenLength Constraints
+     * @return This factory instance (to allow call chaining)
+     * @throws IllegalArgumentException if {@code maxErrorTokenLength} is less than 0
+     *
+     * @since 2.16
+     */
+    public JsonFactory setMaxErrorTokenLength(int maxErrorTokenLength) {
+        if (maxErrorTokenLength < 0) {
+            throw new IllegalArgumentException("maxErrorTokenLength (" + maxErrorTokenLength + ") must be greater than 0");
```

Comment:
Either the condition or the error message is incorrect. I believe the input value must be non-negative, meaning that zero is also acceptable.

Reply:
@cowwoc Right, thank you for finding out! 🙏🏼 Applied.

Reply:
Looks good to me

Reply:
Cheers